src/app/shared/ui-kit/ui-question-card/ui-question-card.component.html

```diff
@@ -0,0 +1,62 @@
+<div class="w-80 space-y-7 rounded-xl bg-success p-4 shadow-xl md:w-[640px]">
+  <div class="flex items-center justify-between">
+    <p>
+      <span class="bg-warning p-1 text-3xl font-semibold text-primary">1</span>
+      <span class="p-1 font-medium">/10</span>
+    </p>
+    <img class="h-20 w-20" [src]="avatarPath" alt="category icon" />
+  </div>
+  <h5 class="text-2xl-plus font-semibold md:text-3xl">
+    Let's begin with this
+    <span class="bg-error p-1">legendary pop group</span>
+    <p>What's their name?</p>
+  </h5>
+
+  <form
+    class="mb-9 flex flex-col flex-wrap items-center justify-center space-y-3"
+  >
+    @for (option of mockOptions; track option.id) {
+      <div class="flex w-40 items-center justify-start">
+        <label
+          [for]="option.id"
+          class="flex cursor-pointer items-center text-black peer-checked:text-bright"
+        >
+          <input
+            type="radio"
+            name="optionRadio"
+            [value]="option.id"
+            [id]="option.id"
+            [checked]="option.checked || null"
+            class="peer hidden"
```

Comment:
It can be because of peer order - https://tailwindcss.com/docs/hover-focus-and-other-states#styling-based-on-sibling-state

Let's rewrite it to work with FormControl and revise the styling, and let's create a separate radio group component